### ISSUE TITLE
bridgetown-website build fails locally

### ISSUE THREAD

**Issue by ayushn21** [bridgetownrb/bridgetown#189]
The build for the bridgetown website is failing on my machine with an error related to `rb2js`.

**Bridgetown Version**:
0.18.6

**To Reproduce**
Clone this repo and then run:
`cd bridgetown-website && yarn webpack-build`

**Current behavior**
The build fails with this error:

```
ERROR in ./frontend/javascript/lib/functions.js.rb
Module build failed (from ./node_modules/rb2js-loader/dist/cjs.js):
Error: /Users/Ayush/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/ruby2js-3.5.2/lib/ruby2js/serializer.rb:366:in `sourcemap': undefined method `expression' for nil:NilClass (NoMethodError)
	from /Users/Ayush/Developer/Personal/open_source/bridgetown/bridgetown-website/rb2js.config.rb:28:in `process_with_source_map'
	from -e:1:in `<main>'

    at ChildProcess.<anonymous> (/Users/Ayush/Developer/Personal/open_source/bridgetown/bridgetown-website/node_modules/rb2js-loader/dist/index.js:57:23)
    at ChildProcess.emit (node:events:329:20)
    at maybeClose (node:internal/child_process:1055:16)
    at Socket.<anonymous> (node:internal/child_process:441:11)
    at Socket.emit (node:events:329:20)
    at Pipe.<anonymous> (node:net:655:12)
 @ ./frontend/javascript/index.js 8:0-74 24:8-22 27:10-24 47:6-23 51:2-19
error Command failed with exit code 2.
```

**Expected behavior**
The website should build successfully.

**Computing environment (please complete the following information):**
 - OS: macOS 10.15.6 (Catalina)
 - Ruby Version: 2.7.2

Any idea what might be wrong? I'm not quite sure where to start with this issue! Thanks :)

**Comment by jaredcwhite** [comment]
There seems to be a regression in the most recent ruby2js update. Will investigate!

**Comment by jaredcwhite** [comment]
@ayushn21 I believe this should be fixed now with the latest 3.5.3 release. Give it a try after a bundle update.

**Comment by ayushn21** [comment]
`bundle update` fixed it. Many thanks @jaredcwhite  :)
